### PR TITLE
Fix tutorial link styles

### LINKED ISSUE #2698: Tutorial link styles
Links are side-by-side with little/no padding, which seems wrong. Also the blue link text is very difficult to read on dark mode blue background

Screenshot from 2020-10-07 12-32-37

### LINKED ISSUE #2711: [overview] tutorial page missing icon
This is how it should look: 

image

### PR DESCRIPTION
**Pull request by bgptr**
Closes #2698 
Closes #2711

### CODE REVIEW

**Review by vctt94 — COMMENTED** [review]
I believe it is not fixing #2711 for me

**Comment by bgptr** [comment]
Should be fine now. Sorry.

**Comment by amassarwi** [comment]
@bgptr thanks for the PR.
Side note: when you're fixing/closing multiple issues you will need to prefix each issue number with closes/fixes.

Something like:
Closes #2698 
Closes #2711

File: app/components/views/HelpPage/TutorialsTab/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { FormattedMessage as T } from "react-intl";
 import { DescriptionHeader } from "layout";
 import { Subtitle } from "shared";
 import { Link } from "react-router-dom";
+import recentTicketsStyles from "../../HomePage/RecentTickets/RecentTickets.module.css";
```
Comment (amassarwi) [inline app/components/views/HelpPage/TutorialsTab/index.js:5]: Please create a local css module and use it here, this breaks the whole idea behind css modules.
css module should be a component specific.

**Comment by bgptr** [comment]
@amassarwi Thanks for the review. I've created a local css module for TutorialsTab and also added the closes prefixes before the issue numbers.

**Review by vctt94 — APPROVED** [review]
great, thanks! LGTM

**Pull request merged**